app/controllers/concerns/exception_handler.rb

```diff
@@ -0,0 +1,32 @@
+module ExceptionHandler
+  extend ActiveSupport::Concern
+
+  class AuthenticationError < StandardError; end
+  class MissingToken < StandardError; end
+  class InvalidToken < StandardError; end
+
+  included do
+    rescue_from ActiveRecord::RecordInvalid, with: :four_twenty_two
```

Comment:
Please be sure to keep the method name more meaningful.
`:four_twenty_two` is not good to use